tiles.util.ts

```diff
@@ -1,146 +1,146 @@
 namespace tiles.util {
     //% fixedInstance jres blockIdentity=images._tile
-    //% tags="tile category-Placeholder"
+    //% tags="tile misc category-Placeholder"
```

Comment:
why the tag change?

Reply:
whoops! I manually changed the tags the first time around and forgot to remove the old one from the json